MediaBrowser.Controller/Collections/CollectionCreatedEventArgs.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using MediaBrowser.Controller.Entities.Movies;
+
+namespace MediaBrowser.Controller.Collections
+{
+    /// <summary>
+    /// Object storing the arguments for the <c>CollectionCreated</c> event.
+    /// </summary>
+    public class CollectionCreatedEventArgs : EventArgs
+    {
+        /// <summary>
+        /// Gets or sets the collection.
+        /// </summary>
+        /// <value>The collection.</value>
```

Comment:
I think don't document `<value>` when it doesn't add anything. This is just a duplicate of the summary.

Reply:
This was existing documentation that was moved to a new file as a result of a refactor. You're suggesting I simply remove the value element?

```suggestion
```